src/router/routes.ts

```diff
@@ -110,7 +118,7 @@ export default (store: Store<State>): RouteRecordRaw[] => {
           name: 'zone-egress-detail-view',
           meta: {
             title: 'Zone Egress',
-            parent: 'zone-egress-list-view',
+            isBreadcrumb: true,
```

Comment:
Despite what I was saying in our more general chat, I think this is definitely in improvement on the previous code.